### PR TITLE
ENSIME proposals (refined after Scala World discussions)

### PR DESCRIPTION
**Pull request by fommil**
I've taken the liberty of adding the following as advisors (or supervisors in some cases). Please confirm if you're ok with this:

- [x] @fommil
- [x] @chipsenkbeil
- [x] @dragos
- [x] @wpopielarski
- [x] @mlangc
- [x] @mutcianm
- [x] @xeno-by
- [x] @sschaef

### CODE REVIEW

**Comment by chipsenkbeil** [comment]
@fommil, don't have permission to check my name, so you'll have to do it for me. ;)

**Comment by mlangc** [comment]
Same for me ;-)

**Comment by wpopielarski** [comment]
@fommil I cannot check my name. Btw I would add @sschaef to this list too or you just talked to him about that and he politely refused :)

**Comment by fommil** [comment]
@wpopielarski good point! @sschaef are you happy to advise on the scala-refactoring project? (or the debugging one?)

**Comment by sschaef** [comment]
sure, but for debugging @wpopielarski is the expert, not me ;)

**Comment by fommil** [comment]
@sschaef yeah, that's what I was thinking. Will add your name to the scala-refactoring one

**Comment by mutcianm** [comment]
Sure, please check for me as I don't have write access

**Comment by fommil** [comment]
awesome! Cross team collaboration FTW :smile:

**Comment by fommil** [comment]
@julienrf this is ready for merge

**Comment by julienrf** [comment]
Thanks @fommil, I’ll review this ASAP!

File: gsoc/2017.md [hunk 1]
```diff
@@ -81,56 +81,96 @@ or Dotty) to identify what can be re-used.
...
+*[Wiesław Popielarski](https://github.com/wpopielarski)* (Scala IDE)
+
+### Future-proof scala-refactoring to use scala.meta
+
+There are three IDEs for Scala: IntelliJ, Scala IDE and ENSIME. Scala
+IDE and ENSIME share a common library for refactorings, e.g. "rename
+method" or "organise imports", but IntelliJ have their own
+implementation.
+
+Within the past year the =scala.meta= / scalafix projects have become
```
Comment (julienrf) [inline gsoc/2017.md]: Why these `=` around “scala.meta”?

Comment (fommil) [inline gsoc/2017.md]: org-mode, should be backticks for markdown

**Comment by julienrf** [comment]
Thanks!

**Comment by propensive** [comment]
Should this be "Scala Sphere", not "Scala World"?

**Comment by fommil** [comment]
@propensive I am always doing that

**Pull request merged**